core/src/main/java/io/grpc/internal/AbstractChannelBuilder.java

```diff
@@ -45,7 +48,8 @@
  *
  * @param <BuilderT> The concrete type of this builder.
  */
-public abstract class AbstractChannelBuilder<BuilderT extends AbstractChannelBuilder<BuilderT>> {
+public abstract class AbstractChannelBuilder<BuilderT extends AbstractChannelBuilder<BuilderT>>
+    extends ChannelBuilder<BuilderT> {
```

Comment:
This is somewhat moot with the introduction of ManagerChannel as part of a broader fixup. That introduces ManagedChannelBuilder which is largely what we want here.